Strategies/ITVision.md

```diff
@@ -0,0 +1,168 @@
+The number of services (or number of systems) is not a concern, the concern is rather the ability to effectively manage technical debt and keep up with demands by making timely, small iterative changes.
+
+ESDC leverages different networks based on its need.
+The Internet for its most common day to day use, reducing bottleneck to its corporate and production network.
+The corporate network for accessing corporate systems (e.g. HR, Finance, and emails), and production network for accessing the systems hosting Canadians' data and proving services to Canadians.
+
+The CIO is seen as a critical partner in service design and delivery, having a seat at the table as much as the Chief Data Office (CDO) and the Service's strategic direction.
+IT's credibility to deliver has been achieved by automating much of its internal development processes when it adopted DevOps.
+
+## 4. Modern tech, technology practices and information stewardship
+
+IT's innovation is focused on solving ESDC's business problems.
+Technologies are experimented with to solve business problems, improving the relationships between IT and Service personnel.
+All of ESDC's IT innovation is released as open source on the Internet to provide its benefits to other federal departments, jurisdictions, and the public.
+
+ESDC enables and promotes innovation in order to keep up with the fast pace of technology changes.
+This is achieved by:
+
+- managing risks in a bi-modal way, allowing space for mode 2 (experiments and innovation) to operate, where a mode 1 (rigorous and based on predictability) is used for production and mission critical systems only
+- providing stable funding to manage support and infrastructure for innovators
+- having well defined processes to provide support in scaling successful experiments into production
+- prioritizing innovation in the organization, using metrics focusing on business objectives, improved pace, and success rates
+
+ESDC is narrowing its technology standards focus on ERP, corporate products allowing economies of scale, and production technology stacks for custom built applications only.
+Only on exceptional cases are other technology standardized (e.g. source code version control).
+Product teams manage their own development and testing tools, as well as experimenting with new technology stacks.
+
+Cyber security has shifted to not only include pre-emptive measures but also invest in resilience capabilities.
+IT solutions managing citizen data are built with a [zero trust architecture](https://www.oreilly.com/library/view/zero-trust-networks/9781491962183/ch01.html).
+ESDC's Cyber security has shifted to protecting data at the source and in transit, instead of relying on perimeter defences.
+Encrypted data is treated as unclassified data, therefore increasing the pool of available public cloud services to chose from.
```

Comment:
Encrypted data is treated as unclassified data, therefore increasing the pool of available public cloud services to chose from.

?? I don't understand the if then relationship between the first part and the second

Reply:
Encrypted data should indeed be treated as unclassified data, as long as it remains encrypted. Systems that decrypt it (even only in memory) still need to be security certified I believe.

Yes, treating encrypted data at rest this way lets us choose from a wide array of unclass public cloud services. However, is that a big opportunity since it will only be for the storage part, not the processing?

Reply:
Propose to remove (still in history) and have side discussion if necessary.

@e-wu